programs/dynamic-bonding-curve/src/state/config.rs

```diff
@@ -635,6 +635,12 @@ impl LiquidityVestingInfo {
         } else {
             0
         };
+        // if liquidity_per_period == 0 (due to precision loss), we would need to adjust number_of_period and frequency to zero
+        // so the vesting is cliff-only lock
+        if liquidity_per_period == 0 {
+            number_of_period = 0;
```

Comment:
If `cliff_duration_from_migration_time == 0`, the `cliff_point` will be `current_timestamp` and trigger error at https://github.com/MeteoraAg/damm-v2/blob/7ad310c90c8d64851aa02524e2127658af9cab8a/programs/cp-amm/src/instructions/ix_lock_position.rs#L42

Reply:
nice catch, pushed fix in commit
https://github.com/MeteoraAg/dynamic-bonding-curve/pull/159/commits/73bc283877619cce4d483025fa77674cf354cc93